worlds/osrs/__init__.py

```diff
@@ -180,21 +192,29 @@ def create_regions(self) -> None:
 
         self.roll_locations()
 
+    def task_within_skill_levels(self, skills_required):
+        # Loop through each required skill. If any of its requirements are out of the defined limit, return false
+        for skill in skills_required:
+            max_level_for_skill = getattr(self.options, f"max_{skill.skill.lower()}_level")
+            if skill.level > max_level_for_skill:
+                return False
+        return True
```

Comment:
You don't actually need to do this, I'm doing this purely as an exercise for myself lol

So this could be...

```py
return all(
    skill.level <= getattr(self.options, f"max_{skill.skill.lower()}_level") for skill in skills_required
)
```

Hey that looks kinda nice, maybe you should do this instead